### DISCUSSION TITLE
Cálculo de PMs de Elfos

### DISCUSSION THREAD

**Discussion by pedrokoii** [YuriAlessandro/gerador-ficha-tormenta20#293] [Problemas]
Os PMs de personagens da raça elfo não estão contabilizando o +1 da Herança Feérica.

![image](https://user-images.githubusercontent.com/47647049/118489289-0e68f600-b6f3-11eb-8c94-6646d8ddb29d.png)

**Comment by YuriAlessandro** [comment]
Valeu pelo feedback, vou dar uma olhada mais tarde e quando corrigir, aviso aqui.

**Comment by YuriAlessandro** [comment]
Demorou um ~pouco~ tempo mas eu corrigi esse problema. Valeu por avisar.
